### PR TITLE
Add ly4k's changes for detecting LSA Isolated Data

### PR DESCRIPTION
**Pull request by rtpt-romankarwacik**
This adds the changes from https://github.com/ly4k/Pypykatz for detecting LSA Isolated Data.

### CODE REVIEW

File: pypykatz/lsadecryptor/packages/msv/decryptor.py [hunk 1]
```diff
@@ -358,6 +367,23 @@ def add_credman_credential(self, credman_credential_entry):
 			
 		self.current_logonsession.credman_creds.append(c)
 		
+	def get_proxy_info(self, msv1_0_module):
+		old_pos = self.reader.tell()
+
+		msv1_0_module = self.reader.reader.get_module_by_name("msv1_0.dll")
+
+		try:
+			position = self.find_signature('msv1_0.dll', b'\xe0zRE}*\xecL\xb2\x14s\x9fAY\xc3\x92')
```
Comment (skelsec) [inline pypykatz/lsadecryptor/packages/msv/decryptor.py:376]: How many LSASS dumps/live machine has this signature been tested against?
Is this always static? I'd hazard to guess it's not.

**Comment by skelsec** [comment]
Hello, thank you for the PR!
Before I can merge it, I'd like to ask a few questions mainly on the testing/signature side:
- have you tested this on all/multiple versions of windows?
- did it pass on all of them?
- would it be possible for you to give sample dumps so I can integrate those in the tests?  
Thank you!

**Comment by rtpt-romankarwacik** [comment]
>have you tested this on all/multiple versions of windows?  did it pass on all of them?
I tested it against Win 10 19H1 18362 w/o PPL/Credential Guard, and Win 11 23H2 22631 with Credential Guard enabled.
For those two it worked.

>would it be possible for you to give sample dumps so I can integrate those in the tests?
I uploaded `credential_guard_win11_23H2_22631.dmp` to the Nextcloud containing the dump with credential guard enabled on Win 11 23H2 22631.

File: pypykatz/lsadecryptor/packages/msv/decryptor.py (hunk 1 above)
Comment (rtpt-romankarwacik) [reply pypykatz/lsadecryptor/packages/msv/decryptor.py:376]: On Win 10 19H1 18362 and Win 11 23H2 22631 it matched, but I'm not sure if it generally holds for all Windows versions using credential guard.

**Comment by rtpt-romankarwacik** [comment]
Small correction, I noticed a dump created by [NativeDump](https://github.com/ricardojoserf/NativeDump) does not contain `msv1_0.dll`, so it does not parse properly. Additionaly, in the minidump library the `CSDVersionRva` is not properly set and also crashes in https://github.com/skelsec/minidump/blob/main/minidump/streams/SystemInfoStream.py#L283, commenting the line out was the workaround I took. Not sure if this is something that should be handled by the library, or if NativeDump should include this info.

**Comment by skelsec** [comment]
Sorry for the delay! I've tested it and it works!
Thank you for your contribution!

**Pull request merged**